pilot/pkg/xds/mesh_network_test.go

```diff
@@ -227,7 +226,7 @@ func TestMeshNetworking(t *testing.T) {
 						Labels: map[string]string{
 							label.TopologyNetwork.Name: "network-2",
 							// set the label here to test it = expectation doesn't change since we map back to that via NodePort
-							annotation.NetworkingGatewayPort.Name: "443",
+							label.NetworkingGatewayPort.Name: "443",
```

Comment:
This change is not related to tunneling. I had to change this line, because `NetworkingGatewayPort` was moved to another package.